astroquery/mast/tesscut.py

```diff
@@ -48,16 +86,21 @@ def __init__(self):
 
         self._TESSCUT_URL = conf.server + "/tesscut/api/v0.1/"
 
-    def get_sectors(self, coordinates, radius=0.2*u.deg):  # pragma: no cover
+    def get_sectors(self, coordinates=None, radius=0.2*u.deg, objectname=None):
         """
         Get a list of the TESS data sectors whose footprints intersect
         with the given search area.
 
         Parameters
         ----------
-        coordinates : str or `astropy.coordinates` object
+        coordinates : str or `astropy.coordinates` object, optional
             The target around which to search. It may be specified as a
             string or as the appropriate `astropy.coordinates` object.
+            One and only one of coordinates and objectname must be supplied.
+        objectname : str, optional
```

Comment:
Could we keep the order of the parameters the same as in the signature?

Reply:
I can do that. I wasn't sure what the best way to handle it was. I think the order in the signature is awkward but necessary to allow for backward compatibility, but went ahead and documented the parameters in the logical order that groups coordinates/objectname together. But maybe consistency is more important in this instance.

Reply:
yes, I agree that it's all way to awkward. I hope once we switch off from python2 we can start using keyword only arguments and then we finally be able to reshuffle the signature the way it's most natural (well, do that at least for new modules, we would still have the issue with backward compatibility for the old ones).